include/zephyr/bluetooth/addr.h

```diff
@@ -288,6 +288,12 @@ int bt_addr_from_str(const char *str, bt_addr_t *addr);
  */
 int bt_addr_le_from_str(const char *str, const char *type, bt_addr_le_t *addr);
 
+void bt_addr_le_copy_resolved(bt_addr_le_t *dst, const bt_addr_le_t *src);
+
+bool bt_addr_le_is_resolved(const bt_addr_le_t *addr);
+
+const bt_addr_le_t *bt_lookup_id_addr(uint8_t id, const bt_addr_le_t *addr);
+
```

Comment:
These should be properly documented, and this change should be a separate commit.

Reply:
Why were these moved to the public header btw?

Reply:
I think this is just a wip push:
https://github.com/zephyrproject-rtos/zephyr/pull/68659#discussion_r1517510533

Reply:
Yes, removed.